README.md

```diff
@@ -69,6 +69,10 @@ This plugin integrates the ServiceNow Virtual Agent in Mattermost. For a stable
 
 ## Connecting to ServiceNow
   - Send any direct message to the `ServiceNow Virtual Agent` bot with the username `servicenow-virtual-agent`.
+    **Note-** If `servicenow-virtual-agent` is not visible in your DMs, click on the plus(+) icon on the right side of "Direct Messages" and search for `servicenow-virtual-agent`.
+
+    ![image](https://user-images.githubusercontent.com/55234496/203471920-abdb02c2-2f79-4882-b3e3-f01da7e9b952.png)
```

Comment:
all other images in the README are of light theme but this one is of dark and looks inconsistent, change it to the light one